webpack.config.js

```diff
@@ -26,6 +36,14 @@ module.exports = {
       }
     ]
   },
+  plugins: [
+    new webpack.optimize.CommonsChunkPlugin("vendor", "js/vendor.bundle.js"),
+    extractCSS,
+    new webpack.optimize.DedupePlugin(),
```

Comment:
This will screw up live watching. The docs say to only use this in production builds.